sickbeard/providers/newznab.py

```diff
@@ -251,6 +254,8 @@ def _make_provider(config):
 
             if len(values) == 9:
                 name, url, key, category_ids, enabled, search_mode, search_fallback, enable_daily, enable_backlog = values
+            elif len(values) == 10:
```

Comment:
Don't think it's meant to work like this. Or it gets 10 values, or the two with name and url.